stable/pipelines/templates/buildplane-config-aws.yaml

```diff
@@ -7,9 +7,9 @@ metadata:
     {{- include "pipelines.labels" . | nindent 4 }}
 data:
   provider: aws
-  accountId: {{ .Values.buildPlane.dynamic.customer.accountId | quote }}
-  nodePoolName: {{ .Values.buildPlane.dynamic.customer.nodePoolName | quote }}
-  nodelimit: {{ .Values.buildPlane.dynamic.customer.nodelimit | quote }}
+  accountId: {{ default "userId" .Values.buildPlane.dynamic.customer.accountId | quote }}
```

Comment:
Why not place the "userId" in the values.yaml instead of using the default here?